README.md

```diff
@@ -18,13 +18,15 @@ Please read about the future of the Firebase Admin PHP SDK on the
 
 ---
 
-- [Installation](#installation)
-  - [Laravel](#laravel)
-- [Configuration](#configuration)
-- [Usage](#usage)
-  - [Multiple projects](#multiple-projects)
-- [Supported Versions](#supported-versions)
-- [License](#license)
+-   [Installation](#installation)
+    -   [Laravel](#laravel)
+-   [Configuration](#configuration)
+    -   [Credentials with JSON files](#credentials-with-json-files-auto-discovered)
+    -   [Credentials with arrays](#credentials-with-arrays)
+-   [Usage](#usage)
+    -   [Multiple projects](#multiple-projects)
+-   [Supported Versions](#supported-versions)
+-   [License](#license)
```

Comment:
```suggestion
- [Installation](#installation)
  - [Laravel](#laravel)
- [Configuration](#configuration)
  - [Credentials with JSON files](#credentials-with-json-files)
  - [Credentials with Arrays](#credentials-with-arrays)
- [Usage](#usage)
  - [Multiple projects](#multiple-projects)
- [Supported Versions](#supported-versions)
- [License](#license)
```

* Restored identation and spacing
* Removed `-auto-discovered` (same with the title in another review comment)
* Capitalized `Arrays`